solidity/call_solana.sol

```diff
@@ -0,0 +1,131 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >= 0.7.0;
+pragma abicoder v2;
+
+interface CallSolana {
```

Comment:
@mnedelchev-vn @sukanyaparashar 
As a Solidity developer, please share your thoughts on this interface.
This is a new precompile contract that would allow running any Solana program from a Solidity contract inside NeonEVM.